File: app/src/main/java/com/smoothradio/radio/model/Category.java
package com.smoothradio.radio.model;

import java.util.List;

public class Category {
    public String label;
    public List<RadioStation> categoryRadioStationList;

    public Category(String label, List<RadioStation> categoryRadioStationList) {
        this.label = label;
        this.categoryRadioStationList = categoryRadioStationList;
    }

}

File: app/src/main/java/com/smoothradio/radio/core/model/Category.kt
package com.smoothradio.radio.core.model;

import java.util.List;

public class Category {
    private final String label;
    private final List<RadioStation> categoryRadioStationList;

    public Category(String label, List<RadioStation> categoryRadioStationList) {
        this.label = label;
        this.categoryRadioStationList = categoryRadioStationList;
    }

    public String getLabel() {
        return label;
    }

    public List<RadioStation> getCategoryRadioStationList() {
        return categoryRadioStationList;
    }
}

